front/lib/api/assistant/generation.ts

```diff
@@ -107,7 +107,7 @@ export async function renderConversationForModel({
       );
       messages.unshift({
         role: "user" as const,
-        name: m.context.username,
+        name: m.context.fullName || m.context.username,
```

Comment:
LGTM can you also look at the metaPrompt we pass the model to make sure it's aligned>?

Reply:
Checked, the meta param is USER_FULL_NAME and indeed is replaced by `m.context.fullName`